app/assets/javascripts/client/inventories/AnalysisModalCtrl.js

```diff
@@ -57,24 +83,37 @@
       vm.alerts.splice(index, 1);
     };
 
-    vm.hasInventories = function () {
+    vm.hasInventories = function() {
       return vm.hasFetchedInventories && _.any(vm.inventories);
     };
 
-    vm.hasNoInventories = function () {
+    vm.hasNoInventories = function() {
       return vm.hasFetchedInventories && _.isEmpty(vm.inventories);
-    }
-    vm.save = function () {
+    };
+
+    vm.save = function() {
       Analysis.create(null, vm.analysis)
           .$promise
-          .then(function (productEntry){
+          .then(function(productEntry) {
             vm.closeModal();
```

Comment:
This might be outside of scope, but creating either an assessments or an inventory redirects the user to it.